docs/guides/designing-the-delta-format.md

```diff
@@ -294,9 +294,9 @@ var change = [
 ]
 ```
 
-Since every character is described, explicit indexes and lengths are no longer necessary. This makes out of order indexes and overlapping ranges impossible to express.
+Since every character is described, explicit indexes and lengths are no longer necessary. This makes indexes out of order and overlapping ranges impossible to express.
 
-From this, we can make the easy optimization to merge adjacent equal Operations, re-introducing length. If the last Operation is a `retain`, we can also simply drop this, since it instructs us to "do nothing to the rest of the document".
+Therefore, we can make the easy optimization to merge adjacent equal Operations, re-introducing _length_. And since the last Operation is a `retain` we can simply drop it, for it simply instructs to "do nothing to the rest of the document".
```

Comment:
'And since' implies that the last Operation is always a `retain` but it's not true. Not sure about the other change either. 'for it simply instructs...'